### ISSUE TITLE
Add job type in tweets

### ISSUE THREAD

**Issue by jace** [hasgeek/hasjob#131]
Let Twitter users decide whether to click through to a job. Prefix or suffix the job type to the tweet as text or a hashtag.

**Comment by jitendravyas** [comment]
Has anyone started working on it?

**Comment by jace** [comment]
No. All the other things we've been adding to the tweet -- location hashtag and employer's twitter handle -- have reduced the number of characters available, so we can't have full-length job types like "Partner for a Venture". We'll need to define a shorter Twitter hashtag for each type and include that.
